spec/message_configuration_server_spec.rb

```diff
@@ -272,4 +272,27 @@
       subject.send(:restart_services)
     end
   end
+
+  describe "#configured?" do
```

Comment:
Can you add a test case for false?

Reply:
Thank you @bdunne. That would indeed be a good idea!

Addressed by commit: https://github.com/ManageIQ/manageiq-appliance_console/pull/154/commits/13a4c579bc5e970afbd502c5a8194e5a3b10f391